### ISSUE TITLE
startIndex darf nicht länger als die Länge der Zeichenfolge sein.

### ISSUE THREAD

**Issue by aladdin007** [phw198/OutlookGoogleCalendarSync#234]
Since approx. two weeks I get following Error Msg and can't sync form my gmail cal to outlook.
```
2017-06-10 16:10:10,762 INFO 5 OutlookGoogleCalendarSync.MainForm [0] - The following error was encountered during sync:-
2017-06-10 16:10:10,997 INFO 5 OutlookGoogleCalendarSync.MainForm [0] - startIndex darf nicht länger als die Länge der Zeichenfolge sein.
Parametername: startIndex
2017-06-10 16:10:10,998 ERROR 5 OutlookGoogleCalendarSync.OGCSexception [0] - System.ArgumentOutOfRangeException: startIndex darf nicht länger als die Länge der Zeichenfolge sein.
Parametername: startIndex
```

[OGcalsync.zip](https://github.com/phw198/OutlookGoogleCalendarSync/files/1065870/OGcalsync.zip)

**Comment by phw198** [comment]
:package: This should hopefully be resolved with [Hotfix-v2.5.0.6.zip](https://github.com/phw198/OutlookGoogleCalendarSync/files/1082580/Hotfix-v2.5.0.6.zip) [[see instructions for applying](https://github.com/phw198/OutlookGoogleCalendarSync/wiki/Reporting-Problems#applying-a-hotfix)].

Please let me know how it goes.

**Comment by marcerickson** [comment]
Thanks.  Hotfix seems to have worked, If I have further issues, will repost.  Thank you.
